docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -256,49 +229,67 @@ so we can see the effects of our changes in real time.
 ```bash
 kubectl logs -f --tail 1 -n otterize-tutorial-kafka-mtls deploy/client
 ```
-This client should be able to communicate with the server:
-```
-Loading mTLS certificates
-Connecting to Kafka
-Creating a producer and a consumer for - mytopic
-Sending messages
-Sent message - Message 1 [sent by client]
-Read message - Message 1 [sent by client]
-Read message - Message 1 [sent by client-other]
-Sent message - Message 2 [sent by client]
-Read message - Message 2 [sent by client-other]
-Read message - Message 2 [sent by client]
-Read message - Message 3 [sent by client-other]
-Sent message - Message 3 [sent by client]
-Read message - Message 3 [sent by client]
-```
+This client should be able to access both topics:
+```
+[client] Loading mTLS certificates
+[client] Connecting to Kafka
+[client] Creating a producer and a consumer for - mytopic
+[client] Sending Messages
+[debug]  [client] Sent message 1 [mytopic]
+[client] Creating a producer and a consumer for - transactions
+[client] Sending messages
+[debug]  [client] Sent message 1 [transactions]
+[client] Sent message to topic: transactions - [client] Sent message 1 [transactions]
+[client] Sent message to topic: mytopic - [client] Sent message 1 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 27 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 18 [mytopic]
+[client] Read message from topic: mytopic - [client] Sent message 2 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 36 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 6 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 33 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 19 [mytopic]
+```
+
+As you can see, both `client` and `client-other` are currently able to access both `mytopic` and `transactions` topics.
```

Comment:
This appears to need moving a few lines below, to after we tail the logs for client-other.